index.js

```diff
@@ -9,8 +9,10 @@ async function run() {
 
 		const header = core.getInput('header');
 		const footer = core.getInput('footer');
-		const includeHash = core.getInput('include-hash');
-		const includeRange = core.getInput('include-range');
+
+		// @TODO: Fix boolean checks when https://github.com/actions/toolkit/issues/361 gets resolved
+		const includeHash = core.getInput('include-hash').toString() === 'true';
+		const includeRange = core.getInput('include-range').toString() === 'true';
```

Comment:
Why toString? Aren’t they already strings?

Reply:
Just in case they decide to make booleans work, or return `null` or `undefined` if the inputs are not passed.

Reply:
Meh, we're bundling code anyway, will drop it.